searx/plugins/hash_plugin.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, plg_cfg: "PluginCfg") -> None:
         self.info = PluginInfo(
             id=self.id,
             name=gettext("Hash plugin"),
-            description=gettext("Converts strings to different hash digests."),
+            description=gettext("Converts strings to different hash digests. Available functions: md5, sha1, sha224, sha256, sha384, sha512."),
```

Comment:
```suggestion
            description=gettext(
                "Converts strings to different hash digests. Available functions: md5, sha1, sha224, sha256, sha384, sha512."
            ),
```
Please wrap this into a new line to fix the CI checks.